src/reset-password/controller/redirect.ts

```diff
@@ -1,8 +1,47 @@
 import Controller from '@curveball/controller';
+import { Context } from '@curveball/core';
+import log from '../../log/service';
+import { EventType } from '../../log/types';
+import * as UserService from '../../user/service';
+import { User } from '../../user/types';
+import { resetPasswordForm } from '../formats/redirect';
 
-class ChangePasswordController extends Controller {
+class ResetPasswordController extends Controller {
 
+  async get(ctx: Context) {
+    if (!ctx.state.session.resetPasswordUser) {
+      throw new Error('The "resetPasswordUser" property must be provided');
```

Comment:
Lets make this a `Forbidden` error. The message should be something that's a bit more user-focused.. maybe something like "You can only use this endpoint after you went to the 'forgot password' flow"